packages/vue-app/template/App.js

```diff
@@ -214,6 +221,24 @@ export default {
 
         return Promise.all(p)
       })
+
+      <% if (features.fetch) { %>
```

Comment:
Should this be considered again here? From #10489

https://github.com/nuxt/nuxt.js/pull/10489/files#diff-45fb7db1f2da36defea7d4cf1f2a5f4db0a8f9f320b889808f204e51e4df8259R201-R202